README.md

```diff
@@ -418,17 +418,25 @@ This macro extracts the string left from the character.
 Usage: 
 `{{ pm_utils.left_from_char('[expression]', '[character]') }}`
 
+#### optional_table ([source](macros/generic/optional_table.sql))
```

Comment:
It makes more sense to add this after the optional macro. This fits lexicographic ordering and you are referring to this macro.

Reply:
I adhered to the ordering that was shown in VS code, but I see that is different from the one in GitHub. I can change it.